packages/flutter/lib/src/material/popup_menu.dart

```diff
@@ -49,18 +69,31 @@ class PopupMenuItem<T> extends StatelessComponent {
         decoration: !hasDivider ? null : new BoxDecoration(
           border: new Border(bottom: new BorderSide(color: theme.dividerColor))
         ),
-        child: new DefaultTextStyle(
-          style: style,
-          child: new Baseline(
-            baseline: _kMenuItemHeight - _kBaselineOffsetFromBottom,
-            child: child
-          )
-        )
+        child: item
       )
     );
   }
 }
 
+class CheckedPopupMenuItem<T> extends PopupMenuItem<T> {
+  CheckedPopupMenuItem({
+    Key key,
+    T value,
+    isChecked: false,
+    bool isDisabled: false,
+    Widget child
+  }) : super(
+    key: key,
+    value: value,
+    isDisabled: isDisabled,
+    child: new ListItem(
+      isDisabled: isDisabled,
+      left: new Icon(icon: isChecked ? 'action/done' : null),
```

Comment:
I would think so although I couldn't find evidence in the MD spec (I've asked about it).

Assuming that the animation is needed, I'd like to make that change in a separate CL, if that's OK.